pod.json

```diff
@@ -12,7 +12,7 @@
           "ports": [{
               "containerPort": 8443,
           }],
-          "command": ["start", "master", "--kubernetes=:8443"],
```

Comment:
@smarterclayton was this pod trying to run against an external kubernetes? if so, I'm not sure how that ever worked with this invocation... just trying to understand intent here